controllers/keystoneapi_controller.go

```diff
@@ -332,10 +332,10 @@ func (r *KeystoneAPIReconciler) reconcileInit(
 	// expose the service (create service, route and return the created endpoint URLs)
 	//
 	var keystonePorts = map[endpoint.Endpoint]endpoint.Data{
-		endpoint.EndpointPublic: endpoint.Data{
+		endpoint.EndpointPublic: {
```

Comment:
sanity check on this one 
it doesn't seem related to linting

Reply:
It is, because in Golang is it is unnecessary to explicitly declare the type of a map value in the context of assigning a value to a key, since the type of the value is already implied by the map declaration itself.

Reply:
thank you for checking